feature/community/src/main/java/com/iteneum/community/data/CommunityViewModel.kt

```diff
@@ -0,0 +1,36 @@
+package com.iteneum.community.data
+
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.*
+import androidx.compose.runtime.State
+import androidx.compose.runtime.mutableStateListOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.snapshots.SnapshotStateList
+import androidx.lifecycle.ViewModel
+import com.iteneum.community.domain.CardMenuItem
+import com.iteneum.community.domain.CardModel
+
+class CommunityViewModel : ViewModel() {
+    private val _communityViewModelState = mutableStateOf(CommunityViewStates())
+    val communityViewModelState: State<CommunityViewStates>
+        get() = _communityViewModelState
+    private val _buttonCards = mutableStateListOf<CardModel>()
+    val buttonCards: SnapshotStateList<CardModel>
+        get() = _buttonCards
+    init {
+        loadButtonCards()
+    }
+    fun loadButtonCards() {
```

Comment:
If this function is not going to be used outside of the ViewModel, change it to private.